src/Autofac.Multitenant/Autofac.Multitenant.csproj

```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <Description>Autofac extension for multitenant application support.</Description>
     <VersionPrefix>5.0.0</VersionPrefix>
-    <TargetFrameworks>net451;netstandard1.1;netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;netstandard2.1;</TargetFrameworks>
```

Comment:
Should probably target net461 as well as netstandard2.0 and 2.1. MS guidance is to target net461 as well as ns2.0 because of some old problems they had with net461.

Reply:
I added `net461` as well.